taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -91,21 +91,33 @@ def build_browsertime_task(config, tasks):
         task["run"]["command"].append("--test={}".format(test_name))
         task["run"]["command"].extend(task.pop("args", []))
 
+        # Setup treherder symbol
+        symbol = test_name
+
+        # taskcluster is merging task attributes with the default ones
+        # resulting the --cold extra option in the ytp warm tasks
+        if "youtube-playback" in task["name"]:
+            task["run"]["command"].remove("--cold")
+            symbol = test_name.replace("youtube-playback-", "ytp-")
+
+        # Setup chimera for combined warm+cold testing
+        if task.pop("chimera", False):
+            task["run"]["command"].append("--chimera")
+
+        # Add '-c' to taskcluster symbol when running cold tests
+        if "--cold" in task["run"]["command"]:
+            symbol += "-c"
+
         # Setup visual metrics
         run_visual_metrics = task.pop("run-visual-metrics", False)
         if run_visual_metrics:
             task["run"]["command"].append("--browsertime-video")
             task["run"]["command"].append("--browsertime-no-ffwindowrecorder")
             task["attributes"]["run-visual-metrics"] = True
+            symbol += "-vismet"
```

Comment:
This will apply the symbol to the test task. We already have code that adds the -vismet to the label in the visual-metrics transform.

Reply:
Removed